tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py34
+envlist = py26,py27,py33,py34
```

Comment:
maybe add pypy, pypy3 too?

Then you can add 

```
'Programming Language :: Python :: Implementation :: PyPy',
```

to classifiers as well :D